packages/design-system/src/DatePicker/DatePicker.tsx

```diff
@@ -71,6 +72,9 @@ function DatePicker(props: DatePickerProps) {
   ) => {
     setSelectedDate(date);
     onChange && onChange(date, e);
+    if (e) {
```

Comment:
do we need this behaviour to be customisable?

Reply:
or is this added coz of erratic behaviour

Reply:
This is added to fix erratic behaviour.